velox/common/base/tests/SimdUtilTest.cpp

```diff
@@ -20,6 +20,7 @@
 #include <gtest/gtest.h>
 
 using namespace facebook::velox;
+using namespace facebook::velox::simd;
```

Comment:
Just call `simd::setAll` in the test case.

Reply:
Done